PowerShell/ScubaGear/baselines/powerplatform.md

```diff
@@ -232,22 +232,22 @@ Non-default environments SHOULD have at least one DLP policy affecting them.
 
 ## 3. Power Platform Tenant Isolation
 
-Power Platform tenant isolation is different from Azure AD-wide tenant
-restriction. It does not impact Azure AD-based access outside of Power
+Power Platform tenant isolation is different from Microsoft Entra wide tenant
+restriction. It does not impact Microsoft Entra -based access outside of Power
```

Comment:
```suggestion
restriction. It does not impact Microsoft Entra-based access outside of Power
```

Just an extra space here and in a few other places listed under `3. Power Platform Tenant Isolation`